site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: MySQL, Backups
 
       - Store MongoDB Backups in Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/store-mongodb-backups-in-minio.md
         Slug: store-mongodb-backups-in-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store MongoDB Backups in MinIO- Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to store MongoDB backups in MinIO Server.
+        SEO Keywords: MongoDB
 
       - Store PostgreSQL Backups in Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/store-postgresql-backups-in-minio.md
         Slug: store-postgresql-backups-in-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store PostgreSQL Backups in MinIO - Cookbook/Recipe
+        SEO Description: In this recipe you will learn how to store PostgreSQL backups in MinIO Server.
+        SEO Keywords: PostgresSQL
 
       - Setup Caddy proxy with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/setup-caddy-proxy-with-minio.md
         Slug: setup-caddy-proxy-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Caddy proxy with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Caddy proxy with MinIO Server.
+        SEO Keywords: Caddy proxy
 
       - Setup Nginx proxy with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/setup-nginx-proxy-with-minio.md
         Slug: setup-nginx-proxy-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Nginx proxy with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Nginx proxy with MinIO Server.
+        SEO Keywords: Nginx proxy
 
       - Store Apache Logs into Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aggregate-apache-logs-with-fluentd-and-minio.md
         Slug: aggregate-apache-logs-with-fluentd-and-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store Apache Logs into MinIO - Cookbook/Recipe
+        SEO Description: In this recipe, we will learn how to use MinIO as log aggregator for fluentd using fluent-plugin-s3 plugin.
+        SEO Keywords: Apache
 
       - Rclone with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/rclone-with-minio.md
         Slug: rclone-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Rclone with MinIO Server- Cookbook/Recipe
+        SEO Description: This recipe describes how to use rclone with MinIO Server.
+        SEO Keywords: Rclone
```

Comment:
We can maybe add some more keywords to indicate that this document is on how to use rclone with MinIO